ambassador/tests/ambassador_test.py

```diff
@@ -394,7 +394,7 @@ def __init__(self, aconf: dict, ir: dict, v1config: dict) -> None:
                     'cluster': filter['cluster']['name']
                 }
 
-                for key in [ 'allowed_headers', 'path_prefix', 'timeout_ms', 'weight' ]:
+                for key in [ 'allowed_request_headers', 'path_prefix', 'timeout_ms', 'weight' ]:
```

Comment:
Not misspelled here. Awesome! :smile:

Reply:
Thanks for pointing it out. This PR would not be the same without it.

Reply:
😆  Any time! :smile: